metron-interface/metron-alerts/e2e/alert-details/alert-status/alert-details-status.e2e-spec.ts

```diff
@@ -23,64 +23,67 @@ import {loadTestData, deleteTestData} from '../../utils/e2e_util';
 import { MetronAlertsPage } from '../../alerts-list/alerts-list.po';
 import {TreeViewPage} from '../../alerts-list/tree-view/tree-view.po';
 
-describe('metron-alerts alert status', function() {
+describe('Test spec for metron details page', function() {
   let page: MetronAlertDetailsPage;
   let listPage: MetronAlertsPage;
   let treePage: TreeViewPage;
   let loginPage: LoginPage;
 
-  beforeAll(() => {
-    loadTestData();
+  beforeAll(async function() : Promise<any> {
     loginPage = new LoginPage();
     listPage = new MetronAlertsPage();
     treePage = new TreeViewPage();
-    loginPage.login();
+
+    await loadTestData();
+    await loginPage.login();
   });
 
-  afterAll(() => {
+  afterAll(async function() : Promise<any> {
     new MetronAlertsPage().navigateTo();
-    loginPage.logout();
-    deleteTestData();
+    await loginPage.logout();
+    await deleteTestData();
   });
 
   beforeEach(() => {
     page = new MetronAlertDetailsPage();
     jasmine.addMatchers(customMatchers);
   });
 
-  it('should change alert statuses', () => {
-    let alertId = 'c4c5e418-3938-099e-bb0d-37028a98dca8';
-
-    page.navigateTo(alertId);
-    page.clickNew();
-    expect(page.getAlertStatus('ANY')).toEqual('NEW');
-    page.clickOpen();
-    expect(page.getAlertStatus('NEW')).toEqual('OPEN');
-    expect(listPage.getAlertStatusById(alertId)).toEqual('OPEN');
-    page.clickDismiss();
-    expect(page.getAlertStatus('OPEN')).toEqual('DISMISS');
-    expect(listPage.getAlertStatusById(alertId)).toEqual('DISMISS');
-    page.clickEscalate();
-    expect(page.getAlertStatus('DISMISS')).toEqual('ESCALATE');
-    expect(listPage.getAlertStatusById(alertId)).toEqual('ESCALATE');
-    page.clickResolve();
-    expect(page.getAlertStatus('ESCALATE')).toEqual('RESOLVE');
-    expect(listPage.getAlertStatusById(alertId)).toEqual('RESOLVE');
-    page.clickNew();
+  xit('should change alert statuses', async function() : Promise<any> {
```

Comment:
Any particular reason to skip this test?

Reply:
This change is from @iraghumitra's original work, so it's probably best he answers this question.